R/data-importer-configuration.R

```diff
@@ -189,71 +188,71 @@ DataImporterConfiguration <- R6::R6Class(
         return(invisible(self))
       }
 
-      mappedColumn <- rClr::clrGet(column, "MappedColumn")
+      mappedColumn <- column$get("MappedColumn")
       if (missing(value)) {
-        errorType <- rClr::clrGet(mappedColumn, "ErrorStdDev")
+        errorType <- mappedColumn$get("ErrorStdDev")
         # The string returned must be mapped to the naming used in DataSet (resp. data repository)
         return(.ImporterErrorTypeToDataSetErrorType[[errorType]])
       }
       validateEnumValue(value, .ImporterErrorTypeToDataSetErrorType)
-      rClr::clrSet(
-        mappedColumn, "ErrorStdDev",
+      mappedColumn$set(
+        "ErrorStdDev",
         getEnumKey(enum = .ImporterErrorTypeToDataSetErrorType, value)
       )
     },
 
     #' @field groupingColumns Column names by which the data will be grouped
     groupingColumns = function(value) {
       if (missing(value)) {
-        return(rClr::clrCall(private$.dataImporterTask, "GetAllGroupingColumns", self$ref))
+        return(private$.dataImporterTask$call("GetAllGroupingColumns", self))
       }
-      private$throwPropertyIsReadonly("groupingColumns")
+      private$.throwPropertyIsReadonly("groupingColumns")
     },
 
     #' @field sheets Names of the sheets (list of strings) of the excel workbook for which the
     #' configuration will be applied.
     sheets = function(value) {
       if (missing(value)) {
-        return(rClr::clrCall(private$.dataImporterTask, "GetAllLoadedSheets", self$ref))
+        return(private$.dataImporterTask$call("GetAllLoadedSheets", self))
       }
       if (length(value) == 0) {
-        rClr::clrCall(self$ref, "ClearLoadedSheets")
+        self$call("ClearLoadedSheets")
         return(invisible(self))
       }
       validateIsString(value)
-      rClr::clrCall(private$.dataImporterTask, "SetAllLoadedSheet", self$ref, value)
```

Comment:
Don' have to pass `ref` any more (in rSharp it would be `NetObject$pointer`, as the code can accept R6 objects that inherit from `NetObject`. This is done here:

https://github.com/Open-Systems-Pharmacology/rSharp/blob/2204ca57c5e8e84c733f3301f4ee9407a76c768f/R/utilities-static.R#L58

and here

https://github.com/Open-Systems-Pharmacology/rSharp/blob/2204ca57c5e8e84c733f3301f4ee9407a76c768f/R/utilities-net-object.R#L80